src/nidmm/metadata/attributes.py

```diff
@@ -292,7 +294,9 @@
         'resettable': 'No',
         'type': 'ViInt32',
 'documentation': {
-'description': 'Returns the minor version number of this instrument driver.',
+'description': '''
+The minor version number of this instrument driver.
```

Comment:
Is it a problem that there's always a new line before and after the content of the documentation? Seems like it shouldn't be there.

Reply:
New lines before and after are always cleaned up before formatting the documentation